index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <link rel="stylesseet" href="styles.css" >
+  </head>
```

Comment:
☺️ I kindly recommend you to use the indentation, this will help your code to be clean and readable, you will also be promoting the use of good practices.
```suggestion
  <head>
    <link rel="stylesseet" href="styles.css" >
  </head>
```